go/vt/vttablet/tabletmanager/vdiff/controller.go

```diff
@@ -141,6 +141,10 @@ func (ct *controller) run(ctx context.Context) {
 		}
 		log.Infof("%s vdiff %s", action, ct.uuid)
 		if err := ct.start(ctx, dbClient); err != nil {
+			if isReadOnlyError(err) {
+				log.Warningf("VDiff %s stopped due to tablet becoming read-only, will retry on new primary", ct.uuid)
```

Comment:
@twthorn / @mattlord: I wonder if we should explain that this may be caused by a reparent, or is that too much information?